scripts/extensions/datetimeField/src/getToggleTemplateDateTimeField.tsx

```diff
@@ -0,0 +1,37 @@
+import {ISuperdesk, IEditorComponentProps} from 'superdesk-api';
+import * as React from 'react';
+import {IDateTimeFieldConfig} from './extension';
+import {Switch} from 'superdesk-ui-framework/react';
+
+interface IPropsAdditional {
+    hideToggle?: boolean;
+}
+
+export function getToggleDateTimeField(superdesk: ISuperdesk) {
+    const {notify} = superdesk.ui;
+
+    return class ToggleDateTimeField
+        extends React.PureComponent<IEditorComponentProps<string | null, IDateTimeFieldConfig> & IPropsAdditional> {
+        render() {
+            const checkbox = (
+                <Switch
+                    value={this.props.value != null}
+                    onChange={(value) => {
+                        if (value) {
+                            const initialConfig = this.props.config.initial_offset_minutes
+                            this.props.setValue(`{{ now|iso_datetime(${initialConfig}) }}`)
+                            notify.success(`Time offset is configured to be ${initialConfig} minutes`)
```

Comment:
not sure we need this, but if we do we should translate it

Reply:
@tomaskikutis suggested that, it would be good to show the message to the user.

Reply:
Could you change this to display a message below the checkbox instead of a notification? And also to use the message from the ticket? It explains how it works in more detail.